src/components/Countdown/Countdown.tsx

```diff
@@ -20,16 +24,35 @@ const Countdown: React.FC<Props> = ({ prizeGivingSecondsRemaining }: Props) => {
       const time = formatDuration(
         intervalToDuration({
           start: new Date(initializedTime),
-          end: new Date(countdownTimer * 1000),
+          end: new Date(countdownTimer),
         }),
         { delimiter: ', ' },
       );
       const split = time.split(', ');
```

Comment:
tweaked this a bit to handle updating of the countdown timer, removed the seconds just to make it easier from a space perspective, and added the case where the countdown is in the past. shout if you'd prefer it done differently :pray:

Reply:
No this is totally cool 👌